unity/Assets/Scripts/QuestEditor/EditorTools.cs

```diff
@@ -0,0 +1,95 @@
+
+    public static void Create()
+    {
+
+        Game game = Game.Get();
+        if (GameObject.FindGameObjectWithTag(Game.DIALOG) != null)
+        {
+            Destroyer.Dialog();
+        }
+
+        // Menu border
+        DialogBox db = new DialogBox(new Vector2(2,2),new Vector2(UIScaler.GetWidthUnits() - 4, UIScaler.GetHeightUnits() - 4), StringKey.NULL);
+        db.AddBorder();
+        db.SetFont(game.gameType.GetHeaderFont());
+
+        TextButton tb = new TextButton(
+            new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 8), new Vector2(10, 2f),
+            VALIDATE_SCENARIO, delegate { Validate_Scenario(); }, Color.white);
+        tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0.03f, 0.0f, 0f);
+        tb.SetFont(game.gameType.GetHeaderFont());
+
+        tb = new TextButton(
+            new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 11), new Vector2(10, 2f),
+            OPTIMIZE_LOCALIZATION, delegate { Optimize_Localization(); }, Color.white);
+        tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0.03f, 0.0f, 0f);
+        tb.SetFont(game.gameType.GetHeaderFont());
+
+        tb = new TextButton(
+            new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 14), new Vector2(10, 2f),
+            EXPORT_LOCALIZATION, delegate { Export_Localization(); }, Color.white);
+        tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0.03f, 0.0f, 0f);
+        tb.SetFont(game.gameType.GetHeaderFont());
+
+        tb = new TextButton(
+            new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 17), new Vector2(10, 2f),
+            IMPORT_LOCALIZATION, delegate { Import_Localization(); }, Color.white);
+        tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0.03f, 0.0f, 0f);
+        tb.SetFont(game.gameType.GetHeaderFont());
+
+        tb = new TextButton(
+            new Vector2((UIScaler.GetWidthUnits() - 10) / 2, 20), new Vector2(10, 2f),
+            CommonStringKeys.CANCEL, delegate { Destroyer.Dialog(); }, Color.white);
+        tb.background.GetComponent<UnityEngine.UI.Image>().color = new Color(0.03f, 0.0f, 0f);
+        tb.SetFont(game.gameType.GetHeaderFont());
+
+        // Menu border
+        DialogBox logBox = new DialogBox(new Vector2(2, 2), new Vector2(UIScaler.GetWidthUnits() - 4, UIScaler.GetHeightUnits() - 4), StringKey.NULL);
+        logBox.AddBorder();
+        logBox.SetFont(game.gameType.GetHeaderFont());
+    }
+
+    /// <summary>
+    /// Check the scenario to find possible bugs (Issue #56)
+    /// </summary>
+    private static void Validate_Scenario()
+    {
+
+    }
+
+    /// <summary>
+    /// Create keys to repeated texts and reference it
+    /// </summary>
+    private static void Optimize_Localization()
+    {
+
+    }
+
+    /// <summary>
+    /// Create a single language localization file to be
+    /// easily translatable. This fila will ignore {qst:*} texts
```

Comment:
Does this mean entries that are only {qst:*}?

I get the idea of this import/export, but I feel that just changing the format on disk to be separate files will be easier for all and importantly make merging and pull requests MUCH easier and more useful.